types/rangy/lib/rangy-highlighter.d.ts

```diff
@@ -0,0 +1,100 @@
+// tslint:disable-next-line:no-self-import
+import * as rangy from 'rangy';
+
+// tslint:disable-next-line:no-declare-current-package
```

Comment:
No tslint rule disabling allowed when doing major change

Reply:
see [comment above](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/34066#discussion_r268049871)

Reply:
also: Almost every tslint rules are disabled in old version (see diff of `tslint.json` file) - include this one